scripts/splitControl.js

```diff
@@ -0,0 +1,129 @@
+/*global $ */
+/* exported initSplit */
+
+function initSplit(newData) {
+    // use these defaults if any newData not set
+    let data = {paneContainer: "#window", verticalSplit: false, splitPercent: 50, minSiz0: 50, minSiz1: 50};
+    for(let key in newData) {
+        data[key] = newData[key];
+    }
+    // paneContainer is the id of a div which will contain three divs:
+    // referred to in this function as pane1, dragBar and pane2.
+    // More split views can be set up within the panes.
+    // minSiz0, minSiz1 minimum allowed size of panes top/left and bottom/right
```

Comment:
`minSizePane1` and `minSizePane2` would be clearer names and also map to how you refer to them later.

Reply:
Yes.